### PR TITLE
Review this to get started

### PR DESCRIPTION
**Pull request by jlkeesey**
[Review on Reviewable](https://reviewable.io/reviews/jlkeesey/reviewninja-welcome/1)

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/jlkeesey/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,41 @@
...
+
+Line notes have flags to indicate what is going on in the code,
+too. Flags are as follows:
+
+`['!fix', '!resolve']`
+- Something in the code needs to be reviewed and fixed before
+  the pull request is merged.
+
+`['!fixed', '!resolved', '!completed']`
+- The issue has been resolved.
```
Comment (jlkeesey) [inline ReadMe.md]: Comment !fix

Comment (jlkeesey) [inline ReadMe.md]: Another comment !fix

File: ReadMe.md [hunk 2]
```diff
@@ -1 +1,41 @@
...
+**Thanks for letting us show you around!**
+
+What you’re seeing here is a GitHub pull request with the code diffs
+showing by default. Feel free to look around.
+
+Over on the right is a panel for line notes that you can connect to a
+line of code in the diff. You can also switch to the conversation
+to see comments on the whole pull request.
+
+Line notes have flags to indicate what is going on in the code,
```
Comment (jlkeesey) [inline ReadMe.md]: Line note

File: ReadMe.md [hunk 3]
```diff
@@ -1 +1,41 @@
...
+automatically.
+
+Another key feature you need to know about is ninja stars. Ninja 
+stars are how you communicate that a pull request is okay to merge. 
+You can set, by repository, how many ninja stars you think pull 
+requests should have before merging.
+
+**To add a ninja star**, click the **ninja star** button.
+
+When your pull request no longer has any open line notes, and has received
```
Comment (jlkeesey) [inline ReadMe.md]: Yay! Done!  !fixed

File: ReadMe.md (hunk 1 above)
Comment (jlkeesey) [inline ReadMe.md]: !resolved

**Pull request merged**